<?xml version="1.0"?>
<launch>
  <node name="BLB_RECORDER" pkg="tta_blb" type="node_videorec.py" respawn="False" output="log"/> 
</launch>